pkg/scheduler/listers/listers.go

```diff
@@ -74,3 +74,36 @@ func GetPodServices(serviceLister v1listers.ServiceLister, pod *v1.Pod) ([]*v1.S
 
 	return services, nil
 }
+
+type podLister []*schedulernodeinfo.NodeInfo
```

Comment:
why does this live here and not with the snapshot? I don't think we will be using it on its own.

Reply:
Yeah, it doesn't need to be here, but it's implementation was independent enough that it could be reused by autoscaler. cc @losipiuk

Reply:
> Yeah, it doesn't need to be here, but it's implementation was independent enough that it could be reused by autoscaler. cc @losipiuk

I don't think we should expose our internal implementation of the interface to any external packages, the whole purpose of defining those interfaces and moving the snapshot to internal is to reduce the surface area of dependency to the minimum. I think we should move it.

Reply:
CA does not need it. We are fine with just having `SharedLister`, `PodLister`, `NodeInforLiser` interfaces exposed.

Reply:
Sounds good. Thanks for the feedback